sms_statistics_service/src/firestore_wrapper.py

```diff
@@ -22,8 +22,8 @@ def __init__(self, cert):
     def _get_active_projects_collection_ref(self):
         return self.client.collection(f"active_projects")
 
-    def _get_sms_stats_doc_ref(self, project_name):
-        return self.client.document(f"metrics/rapid_pro/projects/{project_name}")
+    def _get_sms_stat_doc_ref(self, project_name, iso_string):
```

Comment:
iso_string isn't very clear. Lots of things are 'iso' I'd prefer date_string or date_iso_string

Reply:
Changed to `date_string`